modules/examples/sources/threaded-diskq-source/threaded-diskq-source-grammar.ym

```diff
@@ -85,9 +85,9 @@ threaded_diskq_source_option
       free($3);
     }
   | threaded_source_driver_option
+  | threaded_fetcher_driver_option
```

Comment:
Please add this option to `python_fetcher_option` too.

Reply:
Fixed thanks!